handlers/new.go

```diff
@@ -9,13 +8,15 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"net/url"
+	"os"
 	"path"
 	"strconv"
 
 	"github.com/coreos/go-etcd/etcd"
 )
 
-var baseURI = flag.String("host", "https://discovery.etcd.io", "base location for computed token URI")
+var baseURI = flag.String("host", os.Getenv("DISCOVERY_HOST"), "base location for computed token URI")
```

Comment:
this breaks the default value for `host`. Another way to do it is 1. check env existence 2. set it to this flag is this flag has not been set.